bin/node/runtime/src/lib.rs

```diff
@@ -718,14 +719,30 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 
 parameter_types! {
 	pub const BagThresholds: &'static [u64] = &voter_bags::THRESHOLDS;
+	// TODO: revisit to see if we can generate separate thresholds here
+	pub const BagThresholdsBalance: &'static [u128] = &voter_bags::THRESHOLDS_BALANCES;
 }
 
-impl pallet_bags_list::Config for Runtime {
+type VoterBagsListInstance = pallet_bags_list::Instance1;
+impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
 	type Event = Event;
+	/// The voter bags-list is loosely kept up to date, and the real source of truth for the score
+	/// of each node is the staking pallet.
 	type ScoreProvider = Staking;
-	type WeightInfo = pallet_bags_list::weights::SubstrateWeight<Runtime>;
 	type BagThresholds = BagThresholds;
 	type Score = VoteWeight;
+	type WeightInfo = pallet_bags_list::weights::SubstrateWeight<Runtime>;
+}
+
+type TargetBagsListInstance = pallet_bags_list::Instance2;
```

Comment:
With the current state of the PR, this is not needed. you can ONLY use `UseValidatorsMap` as `type TargetList` as it stands now.

Reply:
You can and should still make the `VoterList` be a new instance with a new prefix and take care of the migration in this PR.

Reply:
I have been looking into this and I'm not sure how the migration should be done. 
1. I don't fully understand how to migrate a defaultInstance prefix to `Instance1` prefix, but I guess that could be researched by running a node locally and comparing the storage diff.
2. It seems like this migration has to be done within the runtime similar to https://github.com/paritytech/substrate/issues/10770#issuecomment-1037481226
and https://substrate.stackexchange.com/questions/3498/how-to-verify-runtime-upgrade-using-try-runtime

Which means we'd only have to implement/run it on those runtimes where we perform the instance rename.